logo.js

```diff
@@ -3693,4 +3693,47 @@ function LogoInterpreter(turtle, stream, savehook)
       throw new Error("Internal error: Unbound procedure");
     return value;
   });
+
+  //----------------------------------------------------------------------
+  //
+  // 9. Sound / Basic Music / Web Speech
+  //
+  //----------------------------------------------------------------------
+
+  var soundSeq = [];
+  var audioCtx = new (window.AudioContext || window.webkitAudioContext)();
+
+  function playSound(frequency, duration) {
+      const oscillator = audioCtx.createOscillator();
```

Comment:
The overall package is compatible with older JS engines, so please stick with `var` instead of `const`.

At some point I'll drop support for older browsers and the codebase can be modernized (async/await, arrow functions, etc) but not yet.